_employment/job-seekers/employment-support.md

```diff
@@ -32,55 +32,55 @@ If you are out of work, you may be eligible for [unemployment support](http://ca
 
         <li>
           <h3>
-            <a href="/employment/job-seekers/one-on-one/">One-on-One Assistance:</a>
+            <a href="/employment/job-seekers/one-on-one/">One-on-One Assistance</a>
           </h3>
         </li>
 
         <li>
           <h3>
-            <a href="https://www.vets.gov/veterans-employment-center/skills-translator">Skills Translator</a>
+            <a href="/employment/job_seekers/skills-translator">Skills Translator</a>
```

Comment:
it should be `job-seekers` with a hyphen. The only thing with an underscore still is `search_jobs` and that really should be changed too, I just haven't had the time to do it.